packages/data-context/src/sources/migration/autoRename.ts

```diff
@@ -115,7 +115,7 @@ export async function getSpecs (projectRoot: string, config: OldCypressConfig):
   const globs = integrationFolder === false
     ? []
     : integrationFolder === 'cypress/integration'
-      ? ['**/*'].map((glob) => `${integrationFolder}/${glob}`)
```

Comment:
why is this necessary? Wont' the integrationTestFiles be sufficient since it returns [`['**/*.{js,ts,jsx,tsx,coffee}']`](https://github.com/cypress-io/cypress/blob/alejandro/fix/select-only-testing-files-for-migration/packages/data-context/src/sources/migration/shouldShowSteps.ts#L17) by default?